lib/galaxy/datatypes/converters/sam_to_bigwig_converter.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="CONVERTER_sam_to_bigwig_0" name="Convert SAM to BigWig" version="1.0.2" hidden="true">
+<tool id="CONVERTER_sam_to_bigwig_0" name="Convert SAM to BigWig" version="1.0.3" hidden="true">
     <requirements>
         <requirement type="package" version="357">ucsc-bedgraphtobigwig</requirement>
-        <requirement type="package" version="1.6">samtools</requirement>
+        <requirement type="package" version="1.18">samtools</requirement>
         <requirement type="package" version="2.27.1">bedtools</requirement>
```

Comment:
Can you also upgrade the secondary dependencies ? I think there's a newer bedtools version.